### DISCUSSION TITLE
What could be a reason for no sound?

### DISCUSSION THREAD

**Discussion by stereokai** [CoderLine/alphaTab#757] [Q&A] [answered]
I tried playing a note like so: `alphaTabApi.playNote(alphaTabApi.tracks[0].staves[0].bars[0].voices[0].beats[0].notes[0])`. However, no sound was played. What should I look into? I'm pretty sure the soundfont file was loaded (otherwise AlphaTab logs an error, doesn't it?). Any help would be appreciated.

**Comment by Danielku15** [comment]
The easiest way to check whether the player is setup and ready, would be to just play the whole file using `.play()`. You could add a listener to the [`playerReady`](https://alphatab.net/docs/reference/api/readyforplayback/) event and log an output there to get verification whether everything is up and running. 

alphaTab will not log an error or warning (as of today) if you try to request play or playNote before everything is initialized. 

Otherwise you could try to prepare a JsFiddle to show the issue.

**Comment by stereokai** [comment, marked as answer]
Thanks for that. I realized I have to explicitly state the sound font URL:
```js
settings.player.soundFont =
      "https://cdn.jsdelivr.net/npm/@coderline/alphatab@latest/dist/soundfont/sonivox.sf2";
```
